node/src/rpc/mod.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020-2022 Manta Network.
+// This file is part of Manta.
+//
+// Manta is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Manta is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Manta.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Parachain-specific RPCs implementation.
+
+use alloc::sync::Arc;
+use core::marker::PhantomData;
+
+mod common;
+mod dolphin;
+
+pub use common::Common;
+pub use dolphin::Dolphin;
+
+/// RPC Extension Type
+pub type RpcExtension = jsonrpc_core::IoHandler<sc_rpc::Metadata>;
+
+/// RPC Extension Builder
+pub struct Builder<C, P, M = common::Common> {
```

Comment:
is common a sensible default here? Shouldn't it be one of the concrete markers like `manta`? Not sure we'd ever want to build a common-only extension